docs/reference/eql/search.asciidoc

```diff
@@ -495,15 +477,7 @@ GET /sec_logs/_eql/search
 ----
 // TEST[s/search/search\?filter_path\=\-\*\.events\.\*fields/]
 
-The API returns the following response. Note the `sort` property of each
-matching event contains an array of two items:
-
-* The first item is the event's <<eql-search-api-timestamp-field,timestamp>>,
-converted to milliseconds since the https://en.wikipedia.org/wiki/Unix_time[Unix
-epoch].
-
-* The second item is the event's `event.id` value. This value is used as a sort
-tiebreaker for events with the same timestamp.
+The API returns the following response.
 
```

Comment:
@jrodewig since I've removed the fields from the results, I had to remove this whole section (the two fields are essentially used only internally and get removed before being returned).

Reply:
That works. I can review the docs after this PR merges.